app/helpers/breadcrumbs_helper.rb

```diff
@@ -11,16 +11,16 @@ def event_breadcrumbs(conference, event, action="")
     end
 
     if action.eql? "edit"
```

Comment:
Agreed, but I'd rather change it after merging the PR, as that is not directly related to the locales, but rather the implementation of the helper.